SpatialGDK/Source/SpatialGDK/Private/Interop/RPCUtils.cpp

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/RPCUtils.h"
+
+#include "Interop/GlobalStateManager.h"
+#include "Interop/SpatialPlayerSpawner.h"
+#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialSender.h"
+#include "Utils/RepLayoutUtils.h"
+#include "Utils/SpatialMetrics.h"
+
+using namespace SpatialGDK;
+
+CommandRequestResult RPCUtils::ExecuteCommand(const FCrossServerRPCParams& Params, USpatialNetDriver* NetDriver)
```

Comment:
This duplicates a lot of code from receiving RPCs in general, right ?

Reply:
that is the idea behind the RPC Utils  :) the overall cleanup isn't completely done yet, because I wanted to wait for Jacques' PR to go in, otherwise we may run into conflicts.